static/app/views/dashboardsV2/widgetCard/widgetQueries.tsx

```diff
@@ -448,9 +449,19 @@ class WidgetQueries extends Component<Props, State> {
           query.columns?.length !== 0
         ) {
           requestData.topEvents = widget.limit ?? TOP_N;
-          // Aggregates need to be in fields as well
           requestData.field = [...query.columns, ...query.aggregates];
 
+          // Compare field and orderby as aliases to ensure requestData has
+          // the orderby selected
+          if (
+            query.orderby &&
+            !requestData.field
```

Comment:
Going to simplify this to `!requestData.field.includes(trimStart(query.orderby, '-'))` because we don't need to compare as aliases anymore